src/app/example_configs/config.yaml

```diff
@@ -22,12 +22,12 @@ entry:
     host: 0.0.0.0
     port: 2337
     ssl:
-      ca_cert: null
-      ca_path: null
+      ca_cert: ''
```

Comment:
I'm not sure these need to change either.

Reply:
I didn't change these manually - these actually change when defaults in `_SPECIFICATION` are updated, which is a neat function.